scripts/spidhub-fargate/environments/uat/storage/config/spid-saml-check/v1/configvolume/spid_users.json

```diff
@@ -0,0 +1,562 @@
+		"email": "luborgia@msn.com",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "cristoforocolombo",
+		"password": "password123",
+		"spidLevel": 2,
+
+		"spidCode": "SPID-005",
+		"familyName": "Colombo",
+		"name": "Cristoforo",
+		"gender": "M",
+		"dateOfBirth": "1942-10-12",
+		"countyOfBirth": "GE",
+		"placeOfBirth": "D969",
+		"idCard": "patenteNautica 102579 capitaneriaPortoGenova 2014-08-05 2024-10-12",
+		"expirationDate": "2018-09-15",
+		"fiscalNumber": "TINIT-CLMCST42R12D969Z",
+		"mobilePhone": "3893893893",
+		"email": "laninalapintaelasantamaria@outlook.com",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "lapulzella",
+		"password": "password123",
+		"spidLevel": 2,
+
+		"spidCode": "SPID-006",
+		"familyName": "D'Arco",
+		"name": "Giovanna",
+		"gender": "F",
+		"dateOfBirth": "1912-01-06",
+		"countyOfBirth": "AO",
+		"placeOfBirth": "A326",
+		"idCard": "cartaIdentita AA121212 comuneAosta 2010-01-13 2020-01-06",
+		"expirationDate": "2019-06-04",
+		"fiscalNumber": "TINIT-DRCGNN12A46A326K",
+		"mobilePhone": "3466433463",
+		"email": "laPulzelladOrléans@yahoo.fr",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "fieramosca",
+		"password": "password123",
+		"spidLevel": 2,
+
+		"spidCode": "SPID-007",
+		"familyName": "Fieramosca",
+		"name": "Ettore",
+		"gender": "M",
+		"dateOfBirth": "1976-08-06",
+		"countyOfBirth": "CE",
+		"placeOfBirth": "B715",
+		"idCard": "patente Z1Z111111Z motorizzazioneBarletta 2017-03-09 2027-08-06",
+		"expirationDate": "2019-08-01",
+		"fiscalNumber": "TINIT-FRMTTR76M06B715E",
+		"mobilePhone": "3323323323",
+		"email": "trediciitalianicontrotredicifrancesi@hotmail.com",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "cleopatra",
+		"password": "password123",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)